packages/cli/src/commands/info.ts

```diff
@@ -19,17 +19,30 @@ import os from 'os';
 import { runPlain } from '../lib/run';
 import { paths } from '../lib/paths';
 import { Lockfile } from '../lib/versioning';
+import fs from 'fs-extra';
 
 export default async () => {
   await new Promise(async () => {
     const yarnVersion = await runPlain('yarn --version');
     // eslint-disable-next-line no-restricted-syntax
-    const isLocal = require('fs').existsSync(paths.resolveOwn('./src'));
+    const isLocal = fs.existsSync(paths.resolveOwn('./src'));
+
+    const backstageFile = paths.resolveTargetRoot('backstage.json');
+    let backstageJson = undefined;
+    if (fs.existsSync(backstageFile)) {
+      const buffer = await fs.readFile(backstageFile);
```

Comment:
Think you can use `fs.readJSON` here instead. And wrap it in a `try/catch` just incase it's invalid 👍

Reply:
Sure, I've made this change, how does that looks to you?